front/components/vaults/VaultResourcesList.tsx

```diff
@@ -275,14 +341,33 @@ export const VaultResourcesList = ({
             isAdmin={isAdmin}
           />
         )}
-        {(category === "folder" || category === "website") && (
-          <EditVaultStaticDataSourcesViews
-            owner={owner}
-            plan={plan}
-            vault={vault}
-            category={category}
-            dataSources={dataSources}
-          />
+        {isWebsiteOrFolder(category) && (
+          <>
+            <EditVaultStaticDataSourcesViews
+              owner={owner}
+              plan={plan}
+              vault={vault}
+              category={category}
+              dataSources={dataSources}
+            />
+            {selectedDataSourceView && (
+              <>
+                <VaultFolderOrWebsiteModal
```

Comment:
We already have these modals in EditVaultStaticDataSourcesViews, can we try to use the same ones ? Maybe remove EditVaultStaticDataSourcesViews and just extract the button ?

Reply:
Sure, I overlooked this component 🙏🏼